scripts/variantstore/wdl/extract/create_cohort_extract_data_table.py

```diff
@@ -235,7 +248,26 @@ def make_extract_table(fq_pet_vet_dataset,
               ):
   try:
     global client
-    default_config = QueryJobConfig(labels={ "id" : f"test_cohort_export_{output_table_prefix}"}, priority="INTERACTIVE", use_query_cache=False )
+    # this is where a set of labels are being created for the cohort extract. Do we want the hardcoded one to be different?
+    query_labels_map = {}
+    query_labels_map["id"]= f"test_cohort_export_{output_table_prefix}"
+    query_labels_map["gvs_tool_name"]= f"create_cohort_export_{output_table_prefix}"
```

Comment:
similarly, let's REMOVE the `{output_table_prefix}` here and have this just be the tool name (which, maybe should be gvs_prepare_callset